src/Core/Components/NavMenu/FluentNavBase.cs

```diff
@@ -70,6 +70,9 @@ public abstract class FluentNavBase : FluentComponentBase
     [CascadingParameter]
     public FluentNavMenu Owner { get; set; } = default!;
 
+    [CascadingParameter]
+    public FluentMenu? OwnerMenu { get; set; }
```

Comment:
`OwnerMenu` is not a good name. Seems to tie the SubMenu to the NavMenu which is not the Case. Maybe just `SubMenu`?